src/shared/components/AddCardForm.js

```diff
@@ -0,0 +1,34 @@
+import React, {PropTypes, Component} from 'react';
+import { addCard } from '../actions/cardList';
+
+class AddCardForm extends Component {
+  render() {
+    let input;
+
+    return (
+      <div>
+        <form onSubmit={e => {
+          e.preventDefault();
+          if (!input.value.trim()) {
+            return;
+          }
+          this.props.dispatch(addCard(input.value));
+          input.value = '';
```

Comment:
For next step, yuyeong will revise overall detailed add-format and add-reducer function.